src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -382,50 +382,72 @@ public void testIsNameInvalidSymbol() {
     @Test
     public void testIsBlank() {
         assertFalse(CommonUtil.isBlank("string"), "Should return false when string is not empty");
+        final int[] input = "string".codePoints().toArray();
```

Comment:
Please move these tests to new test class for `CodePointUtil`

Reply:
>It would also be better to use real input (file text) in util class tests, instead of String values.
<img width="627" alt="Screenshot 2021-12-23 at 11 19 11 AM" src="https://user-images.githubusercontent.com/79860504/147201758-bcf34b62-22c9-4f08-ad84-ca6e0c65aaea.png">
like above

Reply:
Yes, we can probably improve our approach, but let's start here.

Reply:
>Edit: you could also use a real check (such as UnicodeCharactersCheck), this would be even better.

Sorry, I didn't get it.

Reply:
For a given unit test unit test for `CodePointUtil`, you could just do:

```
        final AvoidEscapedUnicodeCharactersCheck check 
               = new AvoidEscapedUnicodeCharactersCheck(); // real check
```

instead of:

```
        final AbstractCheck check = new AbstractCheck() {
            @Override
            public int[] getDefaultTokens() {
                return CommonUtil.EMPTY_INT_ARRAY;
            }

            @Override
            public int[] getAcceptableTokens() {
                return getDefaultTokens();
            }

            @Override
            public int[] getRequiredTokens() {
                return getDefaultTokens();
            }
        };

```

Reply:
Yes, I did it.
But if I am using a file for test inputs instead of a string, then inputs that have whitespace at the end how to consider those. "string         " or "      string     ", the whitespace at the end is neglected.

Reply:
>  the whitespace at the end is neglected.

I do not understand, can you explain with an example? Be aware that many IDE's strip trailing whitespace automatically.

Reply:
"whitespace at end       ", the whitespaces was getting trimmed, no worries it's resolved.

Reply:
done

Reply:
Resolved.